docs/configuration/http_filters/grpc_json_transcoding_filter.rst

```diff
@@ -0,0 +1,19 @@
+.. _config_http_filters_grpc_json_transcoding:
+
+gRPC-JSON transcoding filter
+============================
+
+gRPC :ref:`architecture overview <arch_overview_grpc>`.
+
+This is a filter which enables the bridging of a JSON-REST client to a gRPC server.
+
+.. code-block:: json
+
+  {
+    "type": "both",
+    "name": "grpc_json_transcoding",
+    "config": {
+      "proto_descriptors": "proto.pb",
```

Comment:
I think I'm a little confused about what this filter does. My understanding is that through reflection it can operate in a generic transcoding mode without knowing the pb definition ahead of time. Is that possible? I think many use cases are going to want that. Having to predefine the pb is kind of a pain from operations perspective. Do we predefine the pb so that we can do better error checking? (Maybe just related to needing more documentation).